src/core/server/models/comment/comment.ts

```diff
@@ -263,8 +266,7 @@ export function validateEditable(
 
   // Check to see if the edit window expired.
```

Comment:
the other errors have `todo: better error message` as well, but not sure what this means as the other ones are real errors that should be investigated. Do we just want to send more info along on the other ones?